card.py

```diff
@@ -107,8 +108,8 @@ def check_if_stale(issue, label_name, days_allowed, assigned):
     print("** Checking column \"{}\"".format(column.name))
     if column.name not in COLUMNS.values():
         is_bucket = True
-        print("INFO: unknown column \"{}\" - assuming like Bucket".format(column.name))
-    if column.name == COLUMNS.BUCKET:
+        print("INFO: unknown column \"{}\" - assuming like Bucket".format(column.value))
```

Comment:
I think this should still be name. `column` is the object coming from the API, not an enum. I will correct it.